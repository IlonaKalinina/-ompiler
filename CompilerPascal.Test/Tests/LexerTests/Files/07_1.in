1.1e-12
1.1e12
1.1e+12