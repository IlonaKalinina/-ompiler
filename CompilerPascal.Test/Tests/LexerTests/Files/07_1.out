1 1 REAL 1,1E-12 1.1e-12
2 1 REAL 1,1E+12 1.1e12
3 1 REAL 1,1E+12 1.1e+12
(4, 1) Syntax error
